### PR TITLE
require a revision message on the ClaimsRequestRevision endpoint

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/models/claim.go [hunk 1]
```diff
@@ -87,9 +87,10 @@ type Claim struct {
 	PaymentDate         nulls.Time            `db:"payment_date"`
 	TotalPayout         int                   `db:"total_payout"`
 	LegacyID            nulls.Int             `db:"legacy_id"`
+	RevisionMessage     string                `db:"revision_message" validate:"required_if=Status Revision"`
 	CreatedAt           time.Time             `db:"created_at"`
-	UpdatedAt           time.Time             `db:"updated_at"`
```
Comment (jason-jackson) [inline application/models/claim.go:91]: Why was this moved?

**Review by jason-jackson — APPROVED** [review]

File: application/models/claim.go (hunk 1 above)
Comment (Schparky) [reply application/models/claim.go:91]: I trusted Goland too much. Used the "Create field RevisionMessage" and then the key for "Move Statement Up" to move it up a few lines. Didn't noticed that also moved the blank line.

File: application/models/claim.go [hunk 2]
```diff
@@ -87,6 +87,7 @@ type Claim struct {
 	PaymentDate         nulls.Time            `db:"payment_date"`
 	TotalPayout         int                   `db:"total_payout"`
 	LegacyID            nulls.Int             `db:"legacy_id"`
+	RevisionMessage     string                `db:"revision_message" validate:"required_if=Status Revision"`
```
Comment (fillup) [inline application/models/claim.go:90]: Hmm, do we want a single purpose field like this? I was thinking something like `StatusReason` so it could also apply to coverage `denied` too. It's okay to do it this way, but we may need to add something else to include a message in denials.

**Review by fillup — APPROVED** [review]

**Review by Baggerone — APPROVED** [review]
